recipes/netl-ap-map-flow/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set repo_name = "netl-ap-map-flow" %}
+{% set repo_base_url = "https://github.com/stadelmanma/netl-ap-map-flow" %}
+{% set module_name = "ApertureMapModelTools" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "0c571ac190fe0a69aa94a654f37dc778b487704d4b491144b875e500adc296a6" %}
+{% set py_mod_deps = ['numpy x.x', 'scipy', 'pillow >=3.4.0', 'pyyaml']%}
+
+package:
+  name: {{ repo_name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ repo_name }}-{{ version }}.tar.gz
```

Comment:
I know my tarball was rebuilt and the checksum changed when I renamed my repo but I suppose that would be expected.